app/rpc/websockets/pubsub_api.go

```diff
@@ -400,7 +400,7 @@ func (api *PubSubAPI) subscribePendingTransactions(conn *websocket.Conn) (rpc.ID
 				}
 				api.filtersMu.Unlock()
 
-				if err == websocket.ErrCloseSent {
+				if err != nil {
```

Comment:
is there any other cases that don't need to unsubscribe event ?

Reply:
in common situations, we should unsubscribe all the clients with error